packages/fuel-gauge/src/script-with-configurable.test.ts

```diff
@@ -0,0 +1,91 @@
+import { generateTestWallet } from '@fuel-ts/wallet/test-utils';
+import { readFileSync } from 'fs';
+import type { CoinQuantityLike, WalletUnlocked } from 'fuels';
+import { BN, Script, NativeAssetId, Provider } from 'fuels';
+import { join } from 'path';
+
+import abi from '../test-projects/script-with-configurable/out/debug/script-with-configurable-abi.json';
+
+const bytecode = readFileSync(
+  join(
+    __dirname,
+    '../test-projects/script-with-configurable/out/debug/script-with-configurable.bin'
+  )
+);
```

Comment:
Another place that could reuse the `getForcProject` utility from:
 - https://github.com/FuelLabs/fuels-ts/pull/978